src/components/header/headerSearchbar.tsx

```diff
@@ -0,0 +1,46 @@
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import { Form, Button, InputGroup } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+
+const HeaderSearchbar = () => {
+    let navigate = useNavigate();
```

Comment:
This is not used in the file anymore. Can be removed

Reply:
Done